config/samples/k8s_v1alpha1_nginxingresscontroller.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: k8s.nginx.org/v1alpha1
```

Comment:
I don't know what samples are supposed to be

Reply:
I was a little bit confused because in my mind sample != example...but I guess it's the same?

Reply:
Yeah, that would make sense. I guess that's why they have the kustomization scaffolding as well, to encourage easier editing of examples.

Reply:
yeah it's what goes in the `alm-examples`